src/expr/src/explain/json.rs

```diff
@@ -43,12 +44,33 @@ where
         let sources = self
             .sources
             .iter()
-            .map(|ExplainSource { id, op, .. }| {
-                serde_json::json!({
-                    "id": id,
-                    "op": op
-                })
-            })
+            .map(
+                |ExplainSource {
+                     id,
+                     op,
+                     pushdown_info,
+                 }| {
+                    let mut json = serde_json::json!({
+                        "id": id,
+                        "op": op,
+                    });
+
+                    if let Some(PushdownInfo { trace }) = pushdown_info {
+                        let pushdownable: Vec<_> = trace
+                            .iter()
+                            .enumerate()
+                            .filter(|(_, p)| **p == Pushdownable::Yes)
```

Comment:
I think there is a discrepancy between what you report in the `AS JSON` output an the `AS TEXT` output (the latter seems to also print out items where `**p == Pushdownable::Maybe`).

Reply:
(I think this is addressed by my other comment on this file; let me know if that's not the case!)